GPU/Common/TextureReplacer.cpp

```diff
@@ -783,11 +796,27 @@ void TextureReplacer::NotifyTextureDecoded(const ReplacedTextureDecodeInfo &repl
 	pitch = w * 4;
 
 	SaveTextureTask *task = new SaveTextureTask(std::move(saveBuf));
+
+	task->filename = basePath_ / hashfile;
+	task->saveFilename = newTextureDir_ / hashfile;
+	task->createSaveDirectory = false;
+
+	// Create subfolder as needed.
+#ifdef _WIN32
+	size_t slash = hashfile.find_last_of("/\\");
+#else
+	size_t slash = hashfile.find_last_of("/");
+#endif
+	if (slash != hashfile.npos) {
+		// Does this ever happen?
+		// Create any directory structure as needed.
```

Comment:
An alias could be used to save a texture in a subfolder of `newTextureDir_` (i.e. if you had the hash and purged out your pngs to redump them), although I guess this usage is probably uncommon.

-[Unknown]

Reply:
Oh I see, hm...